package.json

```diff
@@ -1783,7 +1784,8 @@
     "tslint": "^5.20.1",
     "typedoc": "^0.20.25",
     "typescript": "^4.1.5",
-    "vscode": "^1.1.36",
+    "vsce": "^1.87.1",
+    "vscode": "^1.1.37",
```

Comment:
We typically don't like to upgrade the vscode version unless we have to.  We want to be able to run on the lowest possible version.

Reply:
Oh, sorry for that, I upgrade that because when I do yarn install it's failed locally

Reply:
Are you saying that this change requires the update to 1.1.37?

Reply:
That's not necessary, fixed